canvas_modules/common-canvas/src/common-properties/controls/dropdown/dropdown.scss

```diff
@@ -23,6 +23,20 @@
 	}
 
 }
+
+.cds--dropdown{
```

Comment:
This shouldn't use a carbon class at the top level.  This would potentially style components outside of our app.